src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -232,10 +232,13 @@ def load_arguments(self, _):
         c.argument('sql_pool_name', arg_type=name_type, id_part='child_name_1', help='The SQL pool name.')
         c.argument('status', arg_type=get_enum_type(TransparentDataEncryptionStatus),
                    required=True, help='Status of the transparent data encryption.')
+        c.argument('transparent_data_encryption_name', options_list=['--trans-data-encry', '-d'], help='Name of the transparent data encryption.')
 
     # synapse sql pool threat-policy
     with self.argument_context('synapse sql pool threat-policy') as c:
         c.argument('sql_pool_name', arg_type=name_type, id_part='child_name_1', help='The SQL pool name.')
+        c.argument('security_alert_policy_name', options_list=['--sec-alert-ply', '-s'],
```

Comment:
Can we use abbreviation for argument names in CLI? @jsntcy

Reply:
@idear1203, what do you mean? `options_list=['--sec-alert-ply', '-s']` is the abbreviation.

Reply:
fixed

Reply:
@jsntcy sorry for the confusion. Do we allow `--sec-alert-ply` as argument name? Previously I always use full words like `--security-alert-policy`